File: Factorial.java
import java.util.Scanner;

public class Factorial {
    public static int factorial(int n) {
        int result = 1;
        for (int i = 2; i <= n; i++) {
            result *= i;
        }
        return result;
    }
    public static void main(String[] args) {
        System.out.println("Ingrese un numero par sacar su factorial" );
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        System.out.println("Factorial de " + n + " es " + factorial(n));
    }
}

File: Factorial.c
#include <stdio.h>

int factorial(int n) {
    int result = 1;
    for (int i = 2; i <= n; i++) {
        result *= i;
    }
    return result;
}

int main() {
    int n;
    printf("Ingrese el numero para sacar su factorial: ");
    scanf("%d", &n);
    printf("Factorial de %d es %d\n", n, factorial(n));
    return 0;
}